app/config/routing_dev.yml

```diff
@@ -16,3 +16,31 @@ _errors:
 
 _main:
     resource: routing.yml
+
+_admin_stock_hot_reload_json:
+    path:     /stock/{hash}.hot-update.json
+    methods:  [GET]
+    defaults:
+        _controller: PrestaShopBundle:Admin/Stock:hashUpdateJson
+
+_admin_stock_hot_reload_js:
+    path:     /stock/{hash}.hot-update.js
+    methods:  [GET]
+    defaults:
+        _controller: PrestaShopBundle:Admin/Stock:hashUpdateJs
+    requirements:
+        hash: .+
+
+_admin_translations_hot_reload_json:
+    path:     /international/{hash}.hot-update.json
+    methods:  [GET]
+    defaults:
+        _controller: PrestaShopBundle:Admin/Translations:hashUpdateJson
+
+_admin_translations_hot_reload_js:
+    path:     /international/{hash}.hot-update.js
+    methods:  [GET]
+    defaults:
+        _controller: PrestaShopBundle:Admin/Translations:hashUpdateJs
+    requirements:
+        hash: .+
```

Comment:
Are these changes not supposed to be in src/PrestaShopBundle/resources/config/routing_admin.yml?